.github/workflows/deploy-binance.yml

```diff
@@ -0,0 +1,148 @@
+name: Deploy to Binance smart chain
+
+# Run on pushes to master or PRs to master
+on:
+  push:
+    branches:
+      - master
```

Comment:
We need to not forget to change this to main, I already pulled dev into the epic branch.

Reply:
Done :+1: